sevntu-checks/src/test/java/com/github/sevntu/checkstyle/filters/SuppressionAnnotationFilterTest.java

```diff
@@ -0,0 +1,279 @@
+        final String[] suppressed = {
+            "135: Annotation 'Deprecated' should be alone on line.",
+        };
+        verifySuppressed(filterConfig, suppressed);
+    }
+
+    @Test
+    public void testThrowWrongRegexp()
+            throws Exception {
+        final DefaultConfiguration filterConfig =
+                createFilterConfig(SuppressionAnnotationFilter.class);
+        filterConfig.addAttribute("checkNames", "*\\.*");
+        try {
+            createChecker(filterConfig);
+            fail();
+        }
+        catch (CheckstyleException ex) {
+            Assert.assertTrue(ex.getMessage().contains("Cannot set property 'checkNames' to '*\\.*'"));
+        }
+    }
+
+    @Test
+    public void testMisc()
+            throws Exception {
+        final SuppressionAnnotationFilter test = new SuppressionAnnotationFilter();
+        final String[] dummyArray = {
+        };
+        final AuditEvent noMessageEvent = new AuditEvent(test, "abc", null);
+        Assert.assertTrue(test.accept(noMessageEvent));
+        final LocalizedMessage dummyMessage =
+            new LocalizedMessage(0, 0, null, null, dummyArray, null, null, null, null);
+        final AuditEvent noTreeEvent = new AuditEvent(test, "abc", dummyMessage);
+        // Here abortive branch of accept method with no AST present is tested.
+        Assert.assertTrue(test.accept(noTreeEvent));
+    }
+
+    public static DefaultConfiguration createFilterConfig(Class<?> aClass) {
+        return new DefaultConfiguration(aClass.getName());
+    }
+
+    protected void verifySuppressed(Configuration aFilterConfig,
+            String[] aSuppressed)
+                    throws Exception {
+        verify(createChecker(aFilterConfig),
+                getPath("external/InputSuppressionAnnotationtFilter.java"),
+                removeSuppressed(sAllMessages, aSuppressed));
+    }
+
+    @Override
+    protected Checker createChecker(Configuration aFilterConfig)
+            throws Exception {
+        final DefaultConfiguration checkerConfig =
+                new DefaultConfiguration("configuration");
+        final DefaultConfiguration checksConfig = createCheckConfig(TreeWalker.class);
+        checksConfig.addChild(createCheckConfig(DetailAstRootHolder.class));
+        checksConfig.addChild(createCheckConfig(MemberNameCheck.class));
+        checksConfig.addChild(createCheckConfig(MethodNameCheck.class));
+        checksConfig.addChild(createCheckConfig(ParameterNameCheck.class));
+        checksConfig.addChild(createCheckConfig(ConstantNameCheck.class));
+        checksConfig.addChild(createCheckConfig(JavadocMethodCheck.class));
+        checksConfig.addChild(createCheckConfig(AnnotationLocationCheck.class));
+        checkerConfig.addChild(checksConfig);
+        if (aFilterConfig != null) {
+            checkerConfig.addChild(aFilterConfig);
+        }
+        final Checker checker = new Checker();
```

Comment:
From here down, looks like the exact same as `BaseCheckTestSupport.createChecker`, so just call that instead of duplicating it's code.
This would allow you to remove this new method, `getPrintStream`.